en/developer_guide/fieldType.rst

```diff
@@ -0,0 +1,101 @@
+Field type
+==========
+
+Open Orchestra offers a RAD approach to create the Content Types.
+Further information about it are available in the `Content Types documentation`_.
+
+During creating a content type with different fields, you can choose between different types.
+By default Open Orchestra offers few types (text line, date, email, hidden, integer, choice, media, money, rich test, text area).
+
+
+.. image:: ../../images/fieldType.png
+
+Add a field type
+----------------
+To add a new field type you can add in your configuration parameters an entry
+to the ``open_orchestra_backoffice.field_types`` key
+which describes your field type.
+
+The description of your field type must contains these values:
+
+* ``name`` name of your field type
+* ``type`` type of your field, this can be a type natively available in Symfony or a custom type
+* ``label`` translation key of label
+* ``default_value`` (optional) describes the default value field
+* ``options`` (optional) describes options of your field type
+
+``default_value`` and ``options`` are describe in more detail in subsequent sections.
+
+Example of basic field type:
+
+.. code-block:: yaml
+
+    open_orchestra_backoffice.field_types:
+        custom_textarea:
+            label: open_orchestra.form.field_type.textarea
+            type: textarea
+
+Field type options
+------------------
+
+The field type can be configure with options, for instance to add an option
+to field ``custom_textarea`` to configure the max length.
```

Comment:
All you have said is good, but it's a too technical explanation. It requires a more functionnal approach to be clearly understood.
For instance you can say that a text area can have a max length. When adding a text area attribute to a content type, the user should be asked for that max_length. This can be done by describing the max_length option in our conf.